library.js

```diff
@@ -71,6 +77,12 @@ exports.load = function(params, callback) {
 
   router.use(autoLogin);
 
+  router.get('/email_not_found_staging', function (req, res) {
+		res.render('email_not_found_staging');
+  });
+  router.get('/email_not_found', function (req, res) {
```

Comment:
Keep only this route. Let's remove `/email_not_found_staging`. Check other comments below for clarification.